src/DataMiner/SLManagedAutomation/UIBlockDefinition.cs

```diff
@@ -600,6 +600,7 @@ public class UIBlockDefinition
 		/// <remarks>
 		/// <para>It is possible to set a style on some dialog box items.</para>
 		/// <para>The supported styles can be accessed through const strings on the Style class, subdivided per control type (Button, Text, etc.).<br/>
+		/// All Button styles can also be applied to a DownloadButton.<br/>
```

Comment:
Is the casing OK here? I'd only write "DownloadButton" if this refers to a specific object that is spelled that way in the code, but I can't find any mention of "DownloadButton" anywhere. If indeed this does not exist, I'd suggest the following spelling instead:
```suggestion
		/// All button styles can also be applied to a download button.<br/>
```